@xen-orchestra/web-core/lib/components/ui/top-bottom-table/UiTopBottomTable.vue

```diff
@@ -0,0 +1,40 @@
+<!-- v2 -->
+<template>
+  <div class="ui-top-bottom-table">
+    <span class="typo p3-regular label">
+      {{ $t('core.select.n-selected', { count: itemCount, total: itemsCount }) }}
+    </span>
+
+    <UiButton accent="info" size="small" variant="tertiary" @click="emit('toggleSelectAll', true)">
+      {{ $t('core.select.all') }}
+    </UiButton>
+    <UiButton accent="info" size="small" variant="tertiary" @click="emit('toggleSelectAll', false)">
+      {{ $t('core.select.unselect') }}
+    </UiButton>
+  </div>
+</template>
+
+<script setup lang="ts">
+import UiButton from '@core/components/ui/button/UiButton.vue'
+
+defineProps<{
+  itemCount: number
+  itemsCount: number
```

Comment:
What do you think of using clearer names for these props? Something like `selectedItems` and `totalItems` or something like that?

Reply:
Sure